fixtures/demo.json

```diff
@@ -1080,6 +1080,7 @@
       "collection": 1,
       "title": "Placeholder Image",
       "file": "original_images/wagtail.webp",
+      "description": "A Pied Wagtail walking with its left foot in the air. Copyright The Hall of EINAR.",
```

Comment:
I found out that the image is from https://www.thehallofeinar.com/2022/03/pied-wagtail/, and there's a copyright watermark. We should probably replace the image with something in public domain, e.g. from Wikimedia Commons.

Reply:
I've used this one before because it was taken in the UK. The license requires attribution and a link to the source. So we'll have to add those if we decide to use this one: https://commons.wikimedia.org/wiki/File:Wagtail_-_geograph.org.uk_-_6524549.jpg

Reply:
We can find [ones that don't require attribution](https://commons.wikimedia.org/w/index.php?search=wagtail&title=Special:MediaSearch&type=image&haslicense=unrestricted), such as this one: https://commons.wikimedia.org/wiki/File:White_wagtail_(Motacilla_alba).jpg. Using an image with the least restrictive license is probably more important than where the photo was taken.